janus_core/geo_opt.py

```diff
@@ -0,0 +1,55 @@
+"""Geometry optimisation."""
+
+from ase import Atoms
+
+try:
+    from ase.filters import FrechetCellFilter as DefaultFilter
+except ImportError:
+    from ase.constraints import ExpCellFilter as DefaultFilter
+
+from ase.optimize import LBFGS
+
+
+#  pylint: disable=dangerous-default-value
+#  Dictionaries are not modified within the function.
+def optimize(
+    atoms: Atoms,
+    fmax: float = 0.1,
+    dyn_kwargs: dict = {},
```

Comment:
May want to specify specific dict
```suggestion
    dyn_kwargs: dict[str, Any] = {},
```
If supporting pre-3.10, need to use `Dict` from `typing`